apps/src/lib/util/javascriptMode.js

```diff
@@ -42,7 +42,8 @@ export function apiValidateType(
   varName,
   varValue,
   expectedType,
-  opt
+  opt,
+  board
```

Comment:
Would it be better to pass an `isMicroBit` flag or maybe `boardType` so you don't have to pass in the whole `board` object?

Reply:
I actually first wrote this by adding a local constant within each called `isMicroBitBoard` but had to do it multiple times. I'll create a function called `isMicroBitBoard` instead. Thanks!